standard/ERCs/erc-6900.md

```diff
@@ -512,7 +512,17 @@ During execution uninstallation, the account MUST correctly clear flags and othe
 - the account SHOULD call `onUnInstall` on the execution module to initialize the states and track call success if required by user.
 - the account MUST emit `ExecutionUninstalled` as defined in the interface for all uninstalled executions.
 
-### Hook Execution Order
+### Hooks
+
+#### Execution Hooks Data Format
+
+For accounts that implement execution hooks, accounts **MUST** conform to these execution hook formats:
```

Comment:
I think in the rest of the doc, we don't make "MUST" bold, so we should probably removing the bolding here, too. The interpretation of the special all-caps words is mentioned at the top.